guides/common/modules/proc_assigning-an-ansible-role-to-a-host-group.adoc

```diff
@@ -1,7 +1,7 @@
 [id="assigning-an-ansible-role-to-a-host-group_{context}"]
 = Assigning an Ansible Role to a Host Group
 
-You can use Ansible roles for remote management of {EL} versions 8, 7, and 6.9 or later.
+You can use Ansible roles for remote management of {EL} version 8 or later.
```

Comment:
This is talking about management of hosts which is not restricted to RHEL 8 or later. This line should stay the same and be updated (separately) to include EL 9 I believe - @adamruzicka could you confirm?